### ISSUE TITLE
pip install error

### ISSUE THREAD

**Issue by jackytu256** [menpo/menpo#771]
when I install menpo via pip, the error just happens.
```
menpo/transform/piecewiseaffine/fastpwa.cpp:470:27: fatal error: ./fastpwa/pwa.h: No such file or directory
     #include "./fastpwa/pwa.h"
                               ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```
Thanks

**Comment by patricksnape** [comment]
Yes I am also seeing that - that is very strange since we use `pip` for our automated builds - @jabooth - seems that there is an include missing?

**Comment by jabooth** [comment]
@patricksnape hmm that's odd. My first thought was that when [the MAINIFEST.in file was pruned down](https://github.com/menpo/menpo/commit/4c6a172b067c7745bac1b716becb025e206e724c) you removed the only place where that header was getting included - but I believe:
```py
      package_data={'menpo': ['data/*'],
                    '': ['*.pxd', '*.pyx', '*.h', '*.cpp']},
```
in `setup.py` should cover this file. Still, the file isn't getting packaged, so this isn't working as we intended.

Best guess - this `package_data ` listing isn't recursive? And `.pyx` files get included through another mechanism (the fact that they are source files to an `Extension` perhaps)?

**Comment by patricksnape** [comment]
All the `cpp`, `pyx` and `pxd` files are there!?

**Comment by patricksnape** [comment]
But it's the header files are there if I run `sdist` locally. We might need to cut a new release just to bump on pypi. Maybe even `0.7.6.1`?

**Comment by jackytu256** [comment]
Hi @patricksnape,
  I also tried install menpo though conda, but I got a lock issue when feching packages.

 It doesn't work , even though I tried to remove lock file several times. Any though ?

Error message is blow.

` 
LockError: Lock error: Already locked: Lock error: 
LOCKERROR: It looks like conda is already doing something.
The lock [u'/home/jacky/miniconda2/pkgs/menpo-0.7.6-py27_0.tar.bz2.pid6421.conda_lock'] was found. Wait for it to finish before continuing.
If you are sure that conda is not running, remove it and try again.
You can also use: $ conda clean --lock
`
Thanks for your help.

**Comment by patricksnape** [comment]
Are you running multiple conda commands at once? Do you use conda for other environments? I've never seen that before for anything other than building.

**Comment by patricksnape** [comment]
Fixed in 0.7.7 - thanks for reporting!
